client/src/style/_config.scss

```diff
@@ -105,16 +105,19 @@ $tpc-green: #004d57;
   font-weight: 600;
 }
 
-@mixin bold-text() {
+@mixin extra-bold-text() {
   @include text;
   font-weight: 800;
 }
 
+@mixin bold-text() {
+  @include text;
+  font-weight: 700;
+}
+
```

Comment:
Fixed mixins to reflect settings in Figma.
In figma:
- bold is weight 700
- extra bold is wight 800

Also changed any pre-existing occurence of `@include bold` to `@include extra-bold`